pkg/supervisor/supervisor.go

```diff
@@ -82,15 +82,20 @@ func (s *Supervisor) processWaitQuit(ctx context.Context) bool {
 			select {
 			case <-time.After(s.TimeoutStop):
 				continue
-			case <-waitresult:
+			case err := <-waitresult:
+				if err != nil {
+					s.log.WithError(err).Error("Failed to wait for process")
+				} else {
+					s.log.Info("Process exited: ", s.cmd.ProcessState)
+				}
 				return true
 			}
 		}
 	case err := <-waitresult:
 		if err != nil {
 			s.log.WithError(err).Warn("Failed to wait for process")
 		} else {
-			s.log.Warnf("Process exited: %s", s.cmd.ProcessState)
+			s.log.Warnf("Process exited: ", s.cmd.ProcessState)
```

Comment:
Oops. This should have been:

```suggestion
			s.log.Warn("Process exited: ", s.cmd.ProcessState)
```